src/core/biblio-db.js

```diff
@@ -277,20 +220,9 @@ export const biblioDB = {
   async clear() {
     const db = await this.ready;
     const storeNames = [...ALLOWED_TYPES];
-    const stores = await new Promise((resolve, reject) => {
-      const transaction = db.transaction(storeNames, "readwrite");
-      transaction.onerror = () => {
-        reject(
-          new DOMException(transaction.error.message, transaction.error.name)
-        );
-      };
-      resolve(transaction);
-    });
+    const stores = await db.transaction(storeNames, "readwrite");
     const clearStorePromises = storeNames.map(name => {
-      return new Promise(resolve => {
-        const request = stores.objectStore(name).clear();
-        request.onsuccess = resolve;
-      });
+      return stores.objectStore(name).clear();
     });
     Promise.all(clearStorePromises);
```

Comment:
Shouldn't we await this? Looks like a bug.

Reply:
Yes, missed this. I have checked more thoroughly and fixed this at a few more places. Thanks for the catch.

Reply:
Oops... I've seen this come up too I think. Glad you spotted this @saschanaz.